src/components/EntryButton.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { FC } from 'react';
+import styled, { css } from 'styled-components';
+import { EntryButtonProps } from '../types/types';
+import icons from '../assets/svg';
+
+const EntryButton: FC<EntryButtonProps> = ({ visible, onClick }) => {
+  return <Button visible={visible} onClick={onClick} />;
+};
+
+const Button = styled.button<{ visible: boolean }>`
```

Comment:
Компонент выше думаю лишний, он просто скрывает много полей у `styled.button`
```suggestion
const EntryButton = styled.button<{ visible: boolean }>`
```